packages/models-library/src/models_library/api_schemas_webserver/license_goods.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from typing import NamedTuple
+
+from models_library.license_goods import LicenseGoodID, LicenseResourceType
+from models_library.resource_tracker import PricingPlanId
+from pydantic import PositiveInt
+
+from ._base import OutputSchema
+
+
+class LicenseGoodGet(OutputSchema):
```

Comment:
along the same line, I think a "good" is not something that is so much in use. Even chatty confirms it ;)

I guess "licensed item" with then "licensed items" would make more sense. but that is just me and I will not complain more.

Reply:
Yes you are right singular good was probably not the best choice:
My current decision process was:
 - "Products" are already in use, and we use product_name almost everywhere, so that's not an option.
 - "Items" is something I like, but one of the feedbacks I received is that it’s too general. I will wait until everyone shares their opinions and then make the change.
   - "market items"
   - "licensed_items"
   -  ... ~~products~~, goods, items, packages, offerings, assets, ... ? Please help me pick one :)

Reply:
licensed items seems fine. I left some more details in the internal chat